Build failure: xjadeo, a dependency of ardour

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

 nixpkgs on master 
❯ nix build .#ardour     

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
❯ nix build .#ardour                            
error: Cannot build '/nix/store/7w5z7kv2mklm3frwvm42vgx80j3bx4gh-xjadeo-0.8.14.drv'.
       Reason: builder failed with exit code 2.
       Output paths:
         /nix/store/kfb8ldx6yb8d1mh35nk5h21ykc4g68pj-xjadeo-0.8.14
       Last 25 log lines:
       > make[2]: Entering directory '/build/source/src'
       > Making all in xjadeo
       > make[3]: Entering directory '/build/source/src/xjadeo'
       > make  all-am
       > make[4]: Entering directory '/build/source/src/xjadeo'
       > gcc -DHAVE_CONFIG_H -I. -I../..    -Wall -g -O3 -pthread -I/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include -I/nix/store/wj1qf53xjys7kyb5fr8v370dlfmpg31q-freetype-2.13.3-dev/include/freetype2  -I/nix/store/c3g7bdbwg12k104aj4y6720zwq15f3xv-libx11-1.8.12-dev/include -I/nix/store/fz1llaam5z4ix2akx2ix7gjs9lqy21ds-xorgproto-2024.1/include -I/nix/store/gsxchfi6nq7a99pagq02bvb6fwnp31z1-libxpm-3.5.17-dev/include -I/nix/store/kw80frqd6ya2a3763a7pq4b1crmcfh5n-liblo-0.32/include   -I/nix/store/9mikjzccgs273kzcx2rv75yvi3cmgjg6-glu-9.0.3-dev/include -I/nix/store/sv6kfc6f96x5l5h87gbxqvjzmh98cv9v-libglvnd-1.7.0-dev/include -I/nix/store/gzm8lp4fih2shikvzxv9xfj54ps3xag7-libjack2-1.9.22-dev/include "-DSUBVERSION=\"\"" "-DSHAREDIR=\"/nix/store/kfb8ldx6yb8d1mh35nk5h21ykc4g68pj-xjadeo-0.8.14/share\""    -g -O2 -c -o xjadeo-main.o `test -f 'main.c' || echo './'`main.c
       > gcc -DHAVE_CONFIG_H -I. -I../..    -Wall -g -O3 -pthread -I/nix/store/c2dqa8h34ablc8nm7gxlhs4y0b8m50jj-ffmpeg-8.0-dev/include -I/nix/store/wj1qf53xjys7kyb5fr8v370dlfmpg31q-freetype-2.13.3-dev/include/freetype2  -I/nix/store/c3g7bdbwg12k104aj4y6720zwq15f3xv-libx11-1.8.12-dev/include -I/nix/store/fz1llaam5z4ix2akx2ix7gjs9lqy21ds-xorgproto-2024.1/include -I/nix/store/gsxchfi6nq7a99pagq02bvb6fwnp31z1-libxpm-3.5.17-dev/include -I/nix/store/kw80frqd6ya2a3763a7pq4b1crmcfh5n-liblo-0.32/include   -I/nix/store/9mikjzccgs273kzcx2rv75yvi3cmgjg6-glu-9.0.3-dev/include -I/nix/store/sv6kfc6f96x5l5h87gbxqvjzmh98cv9v-libglvnd-1.7.0-dev/include -I/nix/store/gzm8lp4fih2shikvzxv9xfj54ps3xag7-libjack2-1.9.22-dev/include "-DSUBVERSION=\"\"" "-DSHAREDIR=\"/nix/store/kfb8ldx6yb8d1mh35nk5h21ykc4g68pj-xjadeo-0.8.14/share\""    -g -O2 -c -o xjadeo-xjadeo.o `test -f 'xjadeo.c' || echo './'`xjadeo.c
       > xjadeo.c: In function 'event_loop':
       > xjadeo.c:240:25: warning: variable 'we_know_transport_is_not_rolling' set but not used [-Wunused-but-set-variable]
       >   240 |                 uint8_t we_know_transport_is_not_rolling = 0;
       >       |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > xjadeo.c: In function 'index_frames':
       > xjadeo.c:988:45: error: 'AVFrame' has no member named 'pkt_pos'; did you mean 'pkt_dts'?
       >   988 |                 fidx[i].frame_pos = pFrame->pkt_pos;
       >       |                                             ^~~~~~~
       >       |                                             pkt_dts
       > make[4]: *** [Makefile:617: xjadeo-xjadeo.o] Error 1
       > make[4]: Leaving directory '/build/source/src/xjadeo'
       > make[3]: *** [Makefile:461: all] Error 2
       > make[3]: Leaving directory '/build/source/src/xjadeo'
       > make[2]: *** [Makefile:369: all-recursive] Error 1
       > make[2]: Leaving directory '/build/source/src'
       > make[1]: *** [Makefile:529: all-recursive] Error 1
       > make[1]: Leaving directory '/build/source'
       > make: *** [Makefile:425: all] Error 2
       For full logs, run:
         nix log /nix/store/7w5z7kv2mklm3frwvm42vgx80j3bx4gh-xjadeo-0.8.14.drv
error: Cannot build '/nix/store/72lqpd89qh0zjg1za1348lhjpzs010f8-ardour-8.12.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/539jvxhzxb12rqzqp4mhh52sx87a6zii-ardour-8.12
```

### Additional context

```
git log|head
commit 036c3539550f49b625e98fdda29e16cca62040a1
Merge: d9952fba18fe 072902c889fe
Author: Peter Simons <simons@cryp.to>
Date:   Thu Nov 13 06:05:16 2025 +0000

    python3Packages.osc: 1.21.0 -> 1.22.0 (#461074)

```

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.7-cachyos, NixOS, 25.11 (Xantusia), 25.11.20251111.9da7f1c`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/cjjpqrp9xy44j5fjh3mygyxy3i7mslj2-source`



### Notify maintainers

@mitchmindtree @magnetophon @mitchmindtree @ryand56



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/xj/xjadeo/package.nix
+++ b/pkgs/by-name/xj/xjadeo/package.nix
@@ -15,15 +15,15 @@
   xorg,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "xjadeo";
-  version = "0.8.14";
+  version = "0.8.14-unstable-2025-09-30";
 
   src = fetchFromGitHub {
     owner = "x42";
     repo = "xjadeo";
-    rev = "v${version}";
-    sha256 = "sha256-GTg0W3D0BRSxsmeVsB4On3MfwncScEGFJGVJK7wflCM=";
+    rev = "88dbd570148f05956ce9dcc49d4957250d516c7c";
+    hash = "sha256-SFP1uYaPEN9eqB4xaN6V17OGQrOF4ZyBM8NiZ+tSuYY=";
   };
 
   nativeBuildInputs = [
@@ -49,7 +49,7 @@ stdenv.mkDerivation rec {
     portmidi
   ];
 
-  meta = with lib; {
+  meta = {
     description = "X Jack Video Monitor";
     longDescription = ''
       Xjadeo is a software video player that displays a video-clip in sync with
@@ -58,8 +58,8 @@ stdenv.mkDerivation rec {
       synchronizing movie frames with external events.
     '';
     homepage = "https://xjadeo.sourceforge.net";
-    license = licenses.gpl2Plus;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ mitchmindtree ];
+    license = lib.licenses.gpl2Plus;
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [ mitchmindtree ];
   };
 }

